cmd/lncli/commands.go

```diff
@@ -2910,6 +2910,9 @@ func getChanInfo(ctx *cli.Context) error {
 		chanID = ctx.Int64("chan_id")
 	case ctx.Args().Present():
 		chanID, err = strconv.ParseInt(ctx.Args().First(), 10, 64)
+		if err != nil {
+			return err
```

Comment:
Could you add some context to the error like so: `unable to parse chan id`

Reply:
Happily, made the change.